iptables/_rule.sls

```diff
@@ -11,6 +11,9 @@ iptables_{{ chain_name }}_{{ rule_name }}:
   {%- endif %}
   - table: {{ rule.get('table', 'filter') }}
   - chain: {{ chain_name }}
+  {%- if rule.family is defined %}
+  - family: {{ rule.family }}
```

Comment:
Maybe just when simply allowing access to port (eg tcp/22) from anywhere or just referencing interface.. ip6 rule will most likely differ from ip4 only when used with cidr.